lib/Controller/AnalysisController.cpp

```diff
@@ -130,17 +130,18 @@ void AnalysisController::executeVariational() {}
 
 void AnalysisController::executeWholeProgram() {
   size_t ConfigIdx = 0;
-  for (auto _DataFlowAnalysis : DataFlowAnalyses) {
+  for (const auto &DFA : DataFlowAnalyses) {
     std::string AnalysisConfigPath =
         (ConfigIdx < AnalysisConfigs.size()) ? AnalysisConfigs[ConfigIdx] : "";
-    if (std::holds_alternative<DataFlowAnalysisType>(_DataFlowAnalysis)) {
-      auto getTaintConfig = [&]() {
-        if (!AnalysisConfigPath.empty()) {
-          return TaintConfig(IRDB, parseTaintConfig(AnalysisConfigPath));
-        }
-        return TaintConfig(IRDB);
-      };
-      auto DataFlowAnalysis = std::get<DataFlowAnalysisType>(_DataFlowAnalysis);
+    if (std::holds_alternative<DataFlowAnalysisType>(DFA)) {
+      auto getTaintConfig // NOLINT
```

Comment:
What's the problem here?

Reply:
Mhm, do you have a good idea here? The issue is that we store a lambda to a variable which triggers the invalid case styling rule since `getTaintConfig` is actually a variable rather than an ordinary function.

Reply:
So using `GetTaintConfig` as variable name would solve this?